src/types/Guidelines.ts

```diff
@@ -0,0 +1,17 @@
+import { ReactNode } from 'react';
+
+import { IconData } from '@equinor/eds-icons';
+
+interface IconItem {
+  title: string;
+  icon: IconData;
+  color: string;
+  colorBox?: ReactNode;
+}
+
+interface ElementItem {
```

Comment:
I think this file should stay in ACL since that is where the guidelines component is

Reply:
Removed